### ISSUE TITLE
tag: Provide a frequently-used and/or MRU tags list

### ISSUE THREAD

**Issue by unforgettableid** [azatoth/twinkle#1087]
My favorite tags in recent months have been {{undisclosed paid}}, plus various subcategories of {{notability}} tags.

It would be nice if, near the Tag dialog box, you could provide a "Frequently-used tags" or a "Most recently used tags" list. The aforementioned tags could be shown there.

Would I prefer a "frequently-used" list or a "most recently used" list? In my case, I don't think it would really matter. Whichever you prefer.

Thank you for reading this, and thank you for all the work you've put into Twinkle!

**Comment by siddharthvp** [comment]
I don't think we want to do frequently used tags, because that would involve storing a count of the usage of every single tag. 

Maintaining a queue of the most recently used 5 tags and showing these up at the top would be good I guess.

**Comment by Amorymeltzer** [comment]
I don't love storing user data, but now that twinkle actions have a tag, it shouldn't be too hard to get a count of most-popular tags in 1/3/6 months, and then see where we are.  Not a bad idea overall assessment of usage.

**Comment by siddharthvp** [comment]
> I don't love storing user data

User data can be stored in mw.user.options, not publicly, which is how most gadgets, extensions, etc do it.
